t/benchmarks/mailserver_test.go

```diff
@@ -75,10 +76,10 @@ func testMailserverPeer(t *testing.T) {
 	// request messages from mail server
 	symKeyID, err := shhService.AddSymKeyFromPassword(mailServerPass)
 	require.NoError(t, err)
-	ok, err := shhAPI.MarkTrustedPeer(nil, *peerURL)
+	ok, err := shhAPI.MarkTrustedPeer(context.TODO(), *peerURL)
```

Comment:
Do you want to change the context later because it's unclear? Otherwise here and below `context.Background()` would match better.